src/bindings/swig/python/CMakeLists.txt

```diff
@@ -119,6 +119,34 @@ else ()
 		endif ()
 	endif (${SWIG_VERSION} VERSION_LESS "4.0.0")
 
+	# merge module
+	add_cppheaders (HDR_FILES)
+	set_source_files_properties (merge.i PROPERTIES CPLUSPLUS ON)
+	set_source_files_properties (merge.i PROPERTIES SWIG_FLAGS "-py3;-extranative")
+	if (CMAKE_VERSION VERSION_LESS 3.8)
+		swig_add_module (swig-python-merge python merge.i)
+	else (CMAKE_VERSION VERSION_LESS 3.8)
```

Comment:
@markus2330 @kodebach I just copied that from above. But I'm not sure if my eyes deceive me, but the conditions in the if () and else () are exactly the same !?, so none of that code will ever execute on CMAKE >= 3.8